hs/AboutResponse.hs

```diff
@@ -6,17 +6,16 @@ import           Text.Blaze ((!))
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
 import Data.Text.Lazy (Text)
-import Text.Markdown (markdown, def)
 import Happstack.Server
 import MasterTemplate
+import Utilities (mdToHTML)
 
 aboutResponse :: Text -> ServerPart Response
 aboutResponse aboutContents =
    ok $ toResponse $
     masterTemplate "Courseography - About"
                 []
-                (do
-                    header "about"
+                (do header "about"
```

Comment:
OK, but that is inconsistent with every other `*Response.hs` file. Do you want me to change those as well? Should every `do` block change?